lib/statsLogger.js

```diff
@@ -122,6 +123,17 @@ exports.createInstance = function(flushInterval, backend, backendConfig, stats)
   };
 
   /**
+   * Record a stat once per flush, just before the flush is executed.  
+   *
+   * @param name The name of the stat to record
+   * @param statFunction A function that returns the value to record to the stat
+   */
+  StatsLogger.prototype.captureStatOnFlush = function(name, statFunction){
```

Comment:
I considered it, but I don't see any remove methods in use, so I didn't add one.  If you think it's necessary, I'll add it.